.github/PULL_REQUEST_TEMPLATE/feature.md

```diff
@@ -0,0 +1,18 @@
+#### [LEVEL]/[COMPONENT]
+| TASK | <!--- [jira TASK ID](jiraURL) -->             |
+| ---- | --------------------------------------------- |
+| DEMO | <!--- [[LEVEL]/[COMPONENT]](zeit now URL) --> |
+
+##### Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+
+##### Description, Motivation and Context
+<!--- Describe your changes in detail -->
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it is solving an issue... How can it be reproduced in order to compare between both behaviors? -->
+
+##### Screenshots (if appropriate):
```

Comment:
I suggest removing the phrase "if appropriate" as it reduces the perceived value of adding examples of your changes.

As a reinforcement I would add a comment, something like 
<!--- Adding images or gif animations of your changes improves the understanding  of your changes -->